### DISCUSSION TITLE
Can we use the arrow PyCapsule/C api when ingesting from pyarrow.Table?

### DISCUSSION THREAD

**Discussion by timkpaine** [perspective-dev/perspective#2911] [Ideas]
we have a relatively long-winded ingestion from a `pyarrow.Table`:

https://github.com/finos/perspective/blob/768ca4068c79a2c2243aa24fd3a073f09c6dc97d/rust/perspective-python/src/client/python.rs#L166

Even though it would start as a different format, can we save time by going through the pycapsule/c interface: https://arrow.apache.org/docs/format/CDataInterface/PyCapsuleInterface.html#consume-a-pycapsule

xref: https://github.com/finos/perspective/issues/1157

**Comment by texodus** [comment]
Converting to a discussion - definitely interested in higher performance Arrow ingestion!
